controllers/tokens.py

```diff
@@ -9,8 +9,9 @@
 @tokens.route('/tokens', methods=['GET'])
 def send_tokens():
     # parse id param from call
-   tokens = {'Doral':'582EB861-9C13-4C89-B491-15F0AFBF9F47',
+    tokens = {'Hollywood':'C6B3CFCC-C1B2-47B4-9FD3-EEFB2A46E3C6',
```

Comment:
I took out the hollywood token because it didnt give me back any routes and front end was having trouble displaying empty info. Did you find a fix? Or should we keep it deleted?

Reply:
This was a merge conflict, I guess I deleted the wrong line. Will fix now